providers/mailjet/src/lib/mailjet.provider.ts

```diff
@@ -72,4 +74,55 @@ export class MailjetEmailProvider implements IEmailProvider {
       code: CheckIntegrationResponseEnum.SUCCESS,
     };
   }
+  
+  getMessageId(body: any | any[]): string[] {
+    if (Array.isArray(body)) {
+      return body.map((item) => item.id);
```

Comment:
Do the id field exists on the mailjet webhook body?

Reply:
![IMG_20221005_201108.jpg](https://user-images.githubusercontent.com/80174214/194089096-d81fef9d-ddd3-4713-b55e-0031207f7e42.jpg)

The sample event contains message id field, but no response field or attempt field (which are present in SendGrid). Is there any alternative for this?

Reply:
Can we send custom props that are sent back in the webhooks?

Reply:
Hi @davidsoderberg could you please expand on your question?